frame_name: /map

#ros related
input_command_vel_topic_name: /cmd_vel
command_vel_topic_name: /filtered_cmd_vel

#set this to same rate as node that is publishing.  Will gradually lower velocity to 0 if set too high.
command_vel_pub_rate: 20

#set these to reasonable limits to prevent jerkiness
linear_acc_limit: 0.6
angular_acc_limit: 0.6